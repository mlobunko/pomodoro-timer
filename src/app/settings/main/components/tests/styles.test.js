import React from "react";
import { shallow } from "enzyme";
import "enzyme-to-json";

import {
  StyledButtons,
  StyledCenterOfPage,
  StyledHeader,
  StyledListOfSettings,
  StyledMainComponent
} from "../styles";

describe("StyledButtons", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledButtons />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledCenterOfPage", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledCenterOfPage />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledHeader", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledListOfSettings", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledListOfSettings />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe("StyledMainComponent", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<StyledMainComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
