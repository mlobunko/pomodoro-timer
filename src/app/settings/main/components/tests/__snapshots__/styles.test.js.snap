// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledButtons renders correctly 1`] = `
<div
  className="sc-bdVaJa cbrBXw"
/>
`;

exports[`StyledCenterOfPage renders correctly 1`] = `
<div
  className="sc-bwzfXH dObjBL"
/>
`;

exports[`StyledHeader renders correctly 1`] = `
<h1
  className="sc-htpNat keGSwt"
/>
`;

exports[`StyledListOfSettings renders correctly 1`] = `
<div
  className="sc-bxivhb csBXoY"
/>
`;

exports[`StyledMainComponent renders correctly 1`] = `
<div
  className="sc-ifAKCX TuHJi"
/>
`;
